config/templates/package-scripts/prerm.erb

```diff
@@ -51,6 +51,8 @@ when "aix" -%>
   done
 <% when "freebsd" -%>
 # this is a noop on <%= platform_family %>
+<% when "mac_os_x" -%>
```

Comment:
I mainly did this as a temporary measure before I add the launchd service scripts. It will be a separate condition in the next PR.